capa/main.py

```diff
@@ -121,6 +121,11 @@
 logger = logging.getLogger("capa")
 
 
+class FilterConfig(TypedDict, total=False):
+    processes: set[int]
+    functions: set[int]
```

Comment:
```suggestion
    processes: Set[int]
    functions: Set[int]
```

indexing `set` as a type hint was added in Python 3.9, whereas we still support 3.8 for a little longer.
https://docs.python.org/3/library/typing.html#typing.Set

Reply:
I see. I have switched to `typing.Set` now.